### ISSUE TITLE
Class "assetic.controller" does not exist.

### ISSUE THREAD

**Issue by 3rw1n** [sanpii/assetic-bundle#1]
Hi, and thanks for bringing assetic to Symfony 4.

I was trying to remove all deprecations in my code on Symfony 3.4 and replaced symfony's assetic with your fork =>
`
"sanpi/assetic-bundle": "dev-master"`

Works like a charm when I dump my assets in prod, but when in dev, all I see instead of my css/js files is this message =>

```
[exception] 500 | Internal Server Error | InvalidArgumentException
[message] Class "assetic.controller" does not exist.
[1] InvalidArgumentException: Class "assetic.controller" does not exist.
InvalidArgumentException:
Class "assetic.controller" does not exist.

  at vendor\jms\di-extra-bundle\HttpKernel\ControllerResolver.php:63
  at JMS\DiExtraBundle\HttpKernel\ControllerResolver->createController('assetic.controller::render')
     (vendor\symfony\symfony\src\Symfony\Component\HttpKernel\Controller\ControllerResolver.php:88)
  at Symfony\Component\HttpKernel\Controller\ControllerResolver->getController(object(Request))
     (vendor\symfony\symfony\src\Symfony\Component\HttpKernel\Controller\ContainerControllerResolver.php:41)
  at Symfony\Component\HttpKernel\Controller\ContainerControllerResolver->getController(object(Request))
     (vendor\symfony\symfony\src\Symfony\Component\HttpKernel\Controller\TraceableControllerResolver.php:49)
  at Symfony\Component\HttpKernel\Controller\TraceableControllerResolver->getController(object(Request))
     (vendor\symfony\symfony\src\Symfony\Component\HttpKernel\HttpKernel.php:134)
  at Symfony\Component\HttpKernel\HttpKernel->handleRaw(object(Request), 1)
     (vendor\symfony\symfony\src\Symfony\Component\HttpKernel\HttpKernel.php:68)
  at Symfony\Component\HttpKernel\HttpKernel->handle(object(Request), 1, true)
     (vendor\symfony\symfony\src\Symfony\Component\HttpKernel\Kernel.php:200)
  at Symfony\Component\HttpKernel\Kernel->handle(object(Request))
     (web\app_dev.php:30)
```

Does anyone know where I went wrong?

Cheers,
Erwin

**Comment by sanpii** [comment]
> But it doesn't improve.

Yes, this is my point: in production the controller isn’t used. Sorry if I wasn’t clear.

> With "symfony/assetic-bundle": "dev-master", it all works fine.

Do you have a small project example to reproduce the bug?

**Comment by 3rw1n** [comment]
Well bug still exists in my 3.4 version of Symfony, but I decided to try and upgrade to Symfony 4.1 anyway.
In this case it works again so maybe there's no need to investigate more as symfony/assetic-bundle does the job on 3.4 and yours on Symfony 4.

**Comment by sanpii** [comment]
Fixed by #6, thank you.
